docs/src/main/asciidoc/README.adoc

```diff
@@ -6,6 +6,27 @@ include::intro.adoc[]
 
 > NOTE: if you are looking for a way to bind to services then this is the wrong library. Check out the [Spring Cloud Connectors](https://github.com/spring-cloud/spring-cloud-connectors) instead.
 
+= Building documentation
```

Comment:
There's a building section that we usually include:
```
== Building

include::https://raw.githubusercontent.com/spring-cloud/spring-cloud-build/master/docs/src/main/asciidoc/building.adoc[]
```

That would be where to add the building documentation section.